src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -844,8 +862,26 @@ li {
     font-weight: normal;
   }
 
+  // align/center text with checkbox
+  .v-input--checkbox .v-input__control .v-input__slot .v-label.theme--light {
+    padding-top: 2px;
+  }
+
+  .v-input.sp-confirm-label {
+    margin-bottom: 30px !important;
+  }
+
   .theme--light.v-input input, .theme--light.v-input textarea {
     color: $gray9;
   }
 }
+
+.sp-number-tags {
+  margin-bottom: 30px;
+}
+
+.sp-incopr-busi-number-text {
+  margin-left: 6px;
+  color: #495057;
```

Comment:
The margin is for whitespace, right? I think other elements in this UI use 4px or 8px, so you:
- should be consistent with other elements
- can use ml-2 or pl-2

Please use colour variable names exclusively.